<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifier_abt</class>
 <widget class="QDialog" name="modifier_abt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>modifier abonnement</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Desktop/New folder/favicon.ico</normaloff>../../Desktop/New folder/favicon.ico</iconset>
  </property>
  <widget class="QLineEdit" name="TYPE_MOD">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="MOD">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="DATE_INS_MOD">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ID_ABTMOD">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
